.github/actions/setup-rust/action.yml

```diff
@@ -26,6 +26,9 @@ inputs:
   service_account:
     type: string
     required: true
+  targets:
```

Comment:
what sort of format does this expect? can we document that here?

Reply:
does it support CSV? or only space-separated?

Reply:
added description